src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -216,6 +216,7 @@ internal enum MessageID
         IDS_FeatureVarianceSafetyForStaticInterfaceMembers = MessageBase + 12791,
         IDS_FeatureConstantInterpolatedStrings = MessageBase + 12792,
         IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction = MessageBase + 12793,
+        IDS_FeatureLambdaAttributes = MessageBase + 12798,
```

Comment:
curious why the +5 here.

Reply:
> curious why the +5 here.

To reduce the chance of conflict with other features.

---
In reply to: [604497031](https://github.com/dotnet/roslyn/pull/52038#discussion_r604497031) [](ancestors = 604497031)